python-summary-statistics/README.md

```diff
@@ -4,19 +4,41 @@
 
 # Python Statistics
 
-This is a Python implementation of Statistics.
+This is a Python implementation of Statistics. It calculates various summary statistics for entire dataset and
+also for all subgroups created by combining all possible values of nominal covariates.
+
+It has two modes
+
+1. `compute --mode intermediate`
```

Comment:
Could it work like this? Woken would run `compute --mode intermediate` first on all nodes and subsequently run `compute --mode aggregate --job-ids 1 2 3` to combine all the stats.